server/controllers/ToDoController.js

```diff
@@ -0,0 +1,93 @@
+import ToDo from './../models/ToDo';
+
+import Search from './../helpers/Search';
+
+/**
+ * @class ToDoController
+ */
+class ToDoController {
+  /**
+   * Gets all to-dos
+   *
+   * @static
+   *
+   * @param {Object} request - request object
+   * @param {Object} response - response object
+   *
+   * @memberof ToDoController
+   *
+   * @returns {void}
+   */
+  static async getToDos(request, response) {
+    try {
+      const toDos = await Search.searchAll(ToDo, { ownerId: request.user._id });
+
+      const refinedTodos = toDos.map(toDo => ({
+        toDoId: toDo._id,
+        ownerId: toDo.ownerId,
+        title: toDo.title,
+      }));
+
+      response.status(200).json({
+        toDos: refinedTodos,
+      });
+    } catch (error) {
+      response.status(500).json({
+        error: 'Oops! Something broke',
```

Comment:
This message is not really helpful. You can pull the error message out of the `error` object and send it back. It will contain more information.

Reply:
Thank you